#----------------------
# Actions types
#----------------------
int32 GET_CHECKER=0
int32 PUT_CHECKER=1
int32 DROP_CHECKER=2
int32 TOUCH_CHESSBOARD=3

#----------------------
# Control element info
#----------------------
float64 X
float64 Y
int32 Action
